### ISSUE TITLE
Please update for most recent version of WP and Woo

### ISSUE THREAD

**Issue by KoolPal** [woothemes/theme-customisations#26]
Hi,

Please update for most recent version of WP and Woo.

Please do not abandon this like the rest of the outdated code snippets found all over the internet.

Thanks

**Comment by papamantra** [comment]
I agree, its a great plugin that I used all the time but doesn't seem to be compatible with the latest version of wordpress.

Any chance of an update??

Thanks

**Comment by warengonzaga** [comment]
Looking for an update also?
